src/util-base64.c

```diff
@@ -246,7 +246,7 @@ static int B64DecodeCompleteStringWSp(void)
 
     const char *src = "SGVs bG8 gV29y bGQ=";
     const char *fin_str = "Hello World";
-    TEST_RFC2045(src, fin_str, strlen(fin_str) + 1, strlen(fin_str), strlen(src), BASE64_ECODE_OK);
+    TEST_RFC2045(src, fin_str, strlen(fin_str) + 3, strlen(fin_str), strlen(src), BASE64_ECODE_OK);
```

Comment:
> But, 1 is needed in order to have the destination in multiple of 3 else we'll overflow while decoding..

I think we need to avoid overflow even when the destination size is not a multiple of 3